src/hooks/useFlags/flagsmith/useFlagsmithFlags.test.tsx

```diff
@@ -0,0 +1,86 @@
+import { renderHook } from "@testing-library/react-hooks";
+import flagsmith from "flagsmith";
+import { FlagsmithProvider } from "flagsmith/react";
+import React from "react";
+
+import { useFlagsmithFlags } from "./useFlagsmithFlags";
+
+const wrapper = ({ children }) => (
+  <FlagsmithProvider
```

Comment:
same here, please mock it, not render, you are not testing flagsmith, you want to test just your code here